scripts/update_stable.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import re
+
+from common import execute_command, env
+from updater import UpdaterBase
+
+WORKFLOW_PATH = ".github/workflows/get-rust-versions.yml"
+RUSTC_VERSION_RE = re.compile(r"[0-9][.][0-9]{2}[.][0-9]")
```

Comment:
- `[0-9]` == `\d`. And from my point of view, `\d` is more specific since it has special meaning. SO let's better use `\d`
- `[0-9]{2}` - it will break with rustc `1.100`. Of course, it will be only in 4 years, but anyway. Let's better use `\d+`
- `[.]` - not very straightforward way to escape the dot. `\.` is more expected

In summary, I propose the following - `\d+\.\d+\.\d+`